lib/improver/tests/set_up_test_cubes.py

```diff
@@ -257,7 +257,7 @@ def set_up_variable_cube(data, name='air_temperature', units='K',
 
 def set_up_percentile_cube(data, percentiles, name='air_temperature',
                            units='K', spatial_grid='latlon',
-                           percentile_dim_name='percentile_over_realization',
+                           percentile_dim_name='percentile',
```

Comment:
I'd actually be in favour of removing this option completely, but that might require more involved changes and this PR is getting complex enough!  I'll note it on the ticket and I or somebody else can do it when we get around to it.

Reply:
Yes it might be best done in a different PR now.